packages/core/package.nls.json

```
@@ -5,6 +5,8 @@
     "AWS.productName.cn": "Amazon Toolkit",
     "AWS.amazonq.productName": "Amazon Q",
     "AWS.codecatalyst.submenu.title": "Manage CodeCatalyst",
+    "AWS.notifications.title": "Notifications",
+    "AWS.notifications.dismiss.title": "Dismiss",
```

should this be in the `AWS.generic.x` namespace? https://github.com/aws/aws-toolkit-vscode/blob/15c636c764dffbc7d1b7fa7f3b4c024a0b72b108/packages/core/package.nls.json#L239-L241
```suggestion
    "AWS.generic.dismiss": "Dismiss",
```